docs/runners/sagemaker.md

```diff
@@ -0,0 +1,98 @@
+### SageMaker Runner
+
+Leverage [AWS SageMaker](https://aws.amazon.com/sagemaker/) to run your Fondant pipelines.
+
+This makes it easy to scale up your pipelines in a serverless manner without worrying about infrastructure
+deployment.
+
+The Fondant SageMaker runner will compile your pipeline to a SageMaker pipeline spec and submit it to SageMaker.
+
+
+### Installing the SageMaker runner
+
+Make sure to install Fondant with the SageMaker runner extra.
+
+```bash
+pip install fondant[sagemaker]
+```
+
+### Prerequisites
+- You will need a sagemaker domain and user with the correct permissions. You can follow the instructions [here](https://docs.aws.amazon.com/sagemaker/latest/dg/onboard-quick-start.html) to set this up. Make sure to note down the role arn( arn:aws:iam::<account_id>:role/service-role/AmazonSageMaker-ExecutionRole-<creation_timestamp>) of the user you are using since you will need it.
```

Comment:
```suggestion
- You will need a sagemaker domain and user with the correct permissions. You can follow the instructions [here](https://docs.aws.amazon.com/sagemaker/latest/dg/onboard-quick-start.html) to set this up. Make sure to note down the role arn(`arn:aws:iam::<account_id>:role/service-role/AmazonSageMaker-ExecutionRole-<creation_timestamp>`) of the user you are using since you will need it.
```

The `<>` are not shown correctly without the code ticks.